TESTS/assumptions/SPI/SPI.cpp

```diff
@@ -1,3 +1,8 @@
+// check if SPI is supported on this device
+#if !DEVICE_SPI
+    #error SPI is not supported on this platform, add 'DEVICE_SPI' definition to your platform.
```

Comment:
green tea magic as i recall. Cant remember why though. Lets remove it and see what breaks eh?